src/quickpicks/topics.ts

```diff
@@ -0,0 +1,43 @@
+import { ProgressOptions, ThemeColor, ThemeIcon, window } from "vscode";
+import { KafkaTopic } from "../models/topic";
+import { getSchemasForTopicEnv, getTopicsForCluster } from "../viewProviders/topics";
+import { KafkaCluster } from "../models/kafkaCluster";
+import { IconNames } from "../constants";
+
+export async function topicQuickPick(cluster: KafkaCluster): Promise<KafkaTopic | undefined> {
+  const options: ProgressOptions = {
+    location: { viewId: "confluent-topics" },
+    title: "Loading topics...",
+  };
+  return window.withProgress(options, async () => {
+    const topics: KafkaTopic[] = await getTopicsForCluster(cluster, true);
+    if (topics.length == 0) {
+      window.showInformationMessage("No topics found in the cluster " + cluster.name);
+      return;
+    }
+
+    const schemas = await getSchemasForTopicEnv(topics[0]);
```

Comment:
This section smells like could be refactored and have both this use and the main use over in topics view provider calling into same implementation?

In the future.